Source/WebCore/css/CSSValue.cpp

```diff
@@ -100,143 +100,143 @@ DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(CSSValue);
 template<typename Visitor> constexpr decltype(auto) CSSValue::visitDerived(Visitor&& visitor)
 {
     switch (classType()) {
```

Comment:
Would be cleaner to add `using enum ClassType` here and other relevant places so you don't have to prefix all of these with `ClassType::` and can keep it as it.

https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2019/p1099r5.html